### DISCUSSION TITLE
Error SIMNIBS4/Brainstorm3

### DISCUSSION THREAD

**Discussion by andraderenew** [simnibs/simnibs#257] [7.Creating head models]
Dear community:

I would like to report an error that doesn’t allow me to move forward in my analysis. 

I am using SIMNIBS 4 with windows 10, matlab 2023b, brainstorm 3. This is FEM mesh segmentation with SIMNIBS 4 with 15000 vertices and a T1. 

The error is 

C:\Users\andra\SimNIBS-4.0\simnibs_env\lib\site-packages\simnibs\segmentation\samseg\GMM.py:84: RuntimeWarning: invalid value encountered in true_divide

.
.
.

numpy.linalg.LinAlgError: Matrix is not positive definite

status =

	1

.
.
.

SimNIBS call: charm --forceqform —forcerun  Subject 01 path/to/subject_T1.nii

SimNIBS error #1: See command window

Rene Andrade.

**Comment by Kstoffer** [comment]
I have tried to run the segmentation with the same data (as per separate communication) and cannot reproduce the error with the latest version of simNIBS (prerelease v4.1.0a) can you try to run it with this version?

**Comment by oulap** [comment]
Might also be related to this discussion #182

Sometimes the underlying math libraries can cause problems.
